webapp/src/components/App/Singlestudy/explore/Results/ResultDetails/index.tsx

```diff
@@ -67,10 +75,14 @@ function ResultDetails() {
   ) as Array<{ id: string; name: string; label?: string }>;
 
   const filteredItems = useMemo(() => {
-    return items.filter((item) =>
-      isSearchMatching(searchValue, item.label || item.name)
-    );
-  }, [items, searchValue]);
+    return isSynthesis
+      ? synthesisItems.filter((item) =>
```

Comment:
Disable search for synthesis, it's useless

Reply:
Its used to display the synthesis items list